test/data-provider/tree.data-provider.po.ts

```diff
@@ -0,0 +1,69 @@
+export class TreeDataProvider {
```

Comment:
I was watching some examples and I wasn't sure how to name it :)
Removed `po` from file name.